src/engraving/dom/timesig.cpp

```diff
@@ -271,8 +272,12 @@ PropertyValue TimeSig::propertyDefault(Pid id) const
         return PropertyValue::fromValue(Fraction(1, 1));
     case Pid::TIMESIG_TYPE:
         return int(TimeSigType::NORMAL);
+    case Pid::PLACEMENT:
+        return PlacementV::ABOVE;
     case Pid::SCALE:
-        return style().styleV(Sid::timesigScale);
+    {
+        return style().styleV(getPropertyStyle(id)).value<ScaleF>();
+    }
```

Comment:
Isn't this handled by `EngravingItem::propertyDefault` (or rather `EngravingObject::propertyDefault`)?